moto/datapipeline/models.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import unicode_literals
+
+import datetime
+import boto.datapipeline
+from moto.core import BaseBackend
+from .utils import get_random_pipeline_id
+
+
+class PipelineObject(object):
+    def __init__(self, object_id, name, fields):
+        self.object_id = object_id
+        self.name = name
+        self.fields = fields
+
+    def to_json(self):
+        return {
+            "Fields": self.fields,
+            "Id": self.object_id,
+            "Name": self.name,
+        }
+
+
+class Pipeline(object):
+    def __init__(self, name, unique_id):
+        self.name = name
+        self.unique_id = unique_id
+        self.description = ""
+        self.pipeline_id = get_random_pipeline_id()
+        self.creation_time = datetime.datetime.utcnow()
+        self.objects = []
+
+    def to_json(self):
+        return {
+            "Description": self.description,
+            "Fields": [{
+                "key": "@pipelineState",
+                "stringValue": "SCHEDULED"
```

Comment:
I think that in order to be more accurate, the initial state should be `PENDING`, and when `activate()` is called, it becomes `SCHEDULED`. This is a minor and I am happy with the current implementation, but that change would make it more accurate.